samples/react-native/src/Screens/PerformanceScreen.tsx

```diff
@@ -62,6 +65,24 @@ const PerformanceScreen = (props: Props) => {
             props.navigation.navigate('Redux');
           }}
         />
+        <Spacer />
+        <Text>
+          Heavy Page Navigation that navigates back automatically, right after
+          finishing the display.
```

Comment:
`right after finishing the display` -> I thought it's after timeout before recording TTID.

Reply:
The naming is indeed wrong, we return to the previous page before TTID is finished, I changed the text to better clarify it.